Vagrantfile

```diff
@@ -1,7 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-unless Vagrant.has_plugin?("vagrant-service-manager")
+if ENV['SERVICE_MANAGER_TEST'] != 'true' && !Vagrant.has_plugin?("vagrant-service-manager")
```

Comment:
There is no reason to keep it imo. @navidshaikh why do you want to keep it. There are plenty of examples in the feature files?